extensions/huawei/GLSL_HUAWEI_subpass_shading.txt

```diff
@@ -0,0 +1,86 @@
+Name
+
+    HUAWEI_subpass_shading
+
+Name Strings
+
+    GL_HUAWEI_subpass_shading
+
+Contact
+
+    Hueilong Wang (wyvernathuawei), HUAWEI
+
+Contributors
+
+    Hueilong Wang, HUAWEI
+
+Status
+
+    Draft
+
+Version
+
+    Last Modified Date: 2021-04-26
+    Revision: 1
+
+Dependencies
+
+    This extension can be applied to OpenGL GLSL versions 4.60
+    (#version 460) and higher.
+
+    This extension is written against revision 7 of the OpenGL Shading Language
+    version 4.60, dated July 10, 2019.
+
+    This extension interacts with revision 43 of the GL_KHR_vulkan_glsl
+    extension, dated October 25, 2017.
+
+Overview
+
+    This extension allows a subpass shading shader to delcare subpass inputs and use subpass input functions.
+
+Modifications to the OpenGL Shading Language Specification, Version 4.60
+
+    Including the following line in a shader can be used to control the
+    language features described in this extension:
+
+      #extension GL_HUAWEI_subpass_shading                          : <behavior>
+
+    where <behavior> is as specified in section 3.3.
+    New preprocessor #defines are added:
+
+      #define GL_HUAWEI_subpass_shading  
+
+Changes to Chapter 4 of The OpenGL Shading Language Specification, Version 4.60
+
+    Modify Section 4.1.7, (Opaque Types)
+
+    Change the "Subpass-input types are only available in fragment shaders." to 
+
+    Subpass-input types are only available in fragment and subpass shading shaders.
+
+Changes to Chapter 8 of The OpenGL Shading Language Specification, Version 4.60
+
+    Modify Section 8.18, (Subpass-Input Functions)
+
+    Change the "Subpass-input functions are only available when targeting a Vulkan fragment stage." to 
+
+    Subpass-input functions are only available when targeting a Vulkan fragment and subpass shading stage.
```

Comment:
I think this one should be "fragment *or* subpass shading stage"

Reply:
Thanks, I fixed in https://github.com/KhronosGroup/GLSL/pull/163/commits/1114f6bed5d92f19481adecdad26a081fa1b4e09